rumqttd/Cargo.toml

```diff
@@ -1,13 +1,15 @@
 [package]
 name = "rumqttd"
 version = "0.17.0"
-license = "Apache-2.0"
+publish = false
```

Comment:
why is this `publish = false` ? we do publish rumqttd

Reply:
good catch